src/components/PrimaryMap.css

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
Hey, let's run this by Mark D. first before merging to see if it captures what he was thinking.

Reply:
Works for me.